### ISSUE TITLE
train loss

### ISSUE THREAD

**Issue by niliusha123** [idstcv/SoftTriple#5]
when i set pretrained=None, and the epoch is very big, but i find the loss of train can't decrease, How long does the training model take？ and can you share some skill of train or something of loss

**Comment by qian-qi** [comment]
Hi,
Without the pretrained model, you have to increase the learning rate for effective training. However, a pretrained model is important for small date sets to avoid overfitting.

**Comment by niliusha123** [comment]
Hi，thanks for your best contribution
one of your paper's contribution is propose the softTriple loss, so if you or not try some other NN frameworks with softTriple loss and compare the performance

**Comment by qian-qi** [comment]
We tried it on different neural networks but reported the performance on Inception for a fair comparison.
